### DISCUSSION TITLE
Is it not possible to pin UWP apps to Windows 10 start?

### DISCUSSION THREAD

**Discussion by justin13601** [valinet/ExplorerPatcher#1490] [Q&A]
I understand EP has a workaround to pin items to the Windows 10 start. However, I can't pin UWP apps (such as Mail, Calendar, Windows Store Apps, etc.) onto the Windows 10 start menu. When I pin the UWP app shortcut via the explorer dialog opened from "Pin tiles to Windows 10 Start menu from File Explorer", the resulting tile is blank and does nothing. After restarting explorer.exe it disappears. Any ideas?

**Comment by pyrates999** [comment]
Duplicate of https://github.com/valinet/ExplorerPatcher/issues/1108
